trading_calendars/calendar_utils.py

```diff
@@ -30,6 +35,12 @@
     'XSWX': XSWXExchangeCalendar,
     'XTSE': XTSEExchangeCalendar,
     'XTKS': XTKSExchangeCalendar,
+    # Country calendars.
```

Comment:
what is the distinction between country and exchange calendars? the country calendars are also exchange calendars, aren't they?

Reply:
The distinction was that they are all on the same exchange (Euronext) but we needed different calendars for all of them. Furthermore, the intention going in is that these calendars will encompass _all_ the exchanges in each country, so it seemed to make sense to just call them by country name. That being said, having looked at the ISO page of exchanges, there's an argument to be had for distinguishing them as "Euronext Brussels", "Euronext Paris", etc. instead, so maybe we want to do that now? @ssanderson @jmccorriston

Reply:
I think I'd be in favor of calling them Euronext Brussels, etc., since we already do that sort of thing for other exchanges, like aliasing Nasdaq to XNYS